app.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "discord",
+  "scripts": {},
+  "env": {
+    "COMMENT_ATTEMPTS": {
+      "required": true
+    },
+    "COMMENT_WAIT": {
+      "required": true
+    },
+    "NEW_RELIC_APP_NAME": {
+      "required": true
+    },
+    "NEW_RELIC_HIGH_SECURITY": {
+      "required": true
+    },
+    "NODE_ENV": {
+      "required": true
+    },
+    "OAUTH_TOKEN": {
+      "required": true
+    },
+    "TRACKING_ID": {
+      "required": true
+    }
```

Comment:
Heroku says `env` is optional, but doesn't explain what happens when it's missing. Do you happen to know?